src/Serializer/JsonEncoder.php

```diff
@@ -34,7 +34,8 @@ public function __construct(string $format, BaseJsonEncoder $jsonEncoder = null)
 
         // Encode <, >, ', &, and " for RFC4627-compliant JSON, which may also be embedded into HTML.
         $this->jsonEncoder = $jsonEncoder ?: new BaseJsonEncoder(
-            new JsonEncode(JsonResponse::DEFAULT_ENCODING_OPTIONS), new JsonDecode(true)
+        // The value "15" will be replaced by  "JsonResponse::DEFAULT_ENCODING_OPTIONS" when Symfony HttpFoundation 2.7 support will be dropped
```

Comment:
I agree, this was just a workaround but if we could use the php directly why not ? @dunglas WDYT ?